src/main/java/com/github/manolo8/darkbot/core/objects/facades/NpcEventProxy.java

```diff
@@ -0,0 +1,47 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import com.github.manolo8.darkbot.core.utils.ByteUtils;
+import eu.darkbot.api.managers.NpcEventAPI;
+
+import static com.github.manolo8.darkbot.Main.API;
+
+public class NpcEventProxy extends Updatable implements NpcEventAPI {
+    public double time;
+    public String eventID;
+    public String eventName;
+    public String eventDescriptionID;
+    public String eventDescription;
+    public String npc_leftStatus;
+    public double leftToKill;
+    public double bossCount;
+    public boolean eventActive;
+
+    @Override
+    public double getRemainingTime() {
+        return time;
+    }
+
+    @Override
+    public NpcEventAPI.Status getStatus() {
+        if(this.eventActive){
+            return Status.ACTIVE;
+        }else{
+            return Status.INACTIVE;
+        } //need to detect when event is over;
```

Comment:
```suggestion
        if (this.eventActive) {
            return Status.ACTIVE;
        } else {
            return Status.INACTIVE;
        } //need to detect when event is over;
```
please fix spacing, also consider if we can only detect active/inactive potentially making it a boolean (ideally tho, we'd like to be able to distinguish that)

Reply:
This was to keep consistency with other proxies